### DISCUSSION TITLE
Week 5 quantum circuit simulation

### DISCUSSION THREAD

**Discussion by Jay5454** [chpc-tech-eval/scc#218] [General]
@nyameko I was going through the some of the other team's task and seems like for the team doing quantum circuit simulation week 5, you assigned to them the same work as CPU freq and scaling, was it an unexpected mistake maybe?

**Comment by nyameko** [comment]
[FIXED](https://github.com/chpc-tech-eval/scc/issues/185)

Thanks @Jay5454 for catching the typo and assisting us in maintaining the content!

We (@MHashatsi & @mpofukelvintafadzwa) appreciate your contribution 😄
